examples/laundry-washer-app/nxp/common/main/DeviceCallbacks.cpp

```diff
@@ -59,7 +59,7 @@ void OnTriggerEffect(::Identify * identify)
 }
 
 Identify gIdentify0 = {
-    chip::EndpointId{ 1 },
+    chip::EndpointId{ 0 },
```

Comment:
Identify is not generally allowed on endpoint 0.

Reply:
Thanks for the comment, updated